app/helpers/sessions_helper.rb

```diff
@@ -11,4 +11,9 @@ def logged_in?
     !current_user.nil?
   end
 
+  def log_out
```

Comment:
This method is called in the previous commit. 

(Its possible that github is reordering the commits in a weird order... it does that sometimes)